dev/io.openliberty.microprofile.jwt.2.1.internal_fat_tck/publish/servers/jwt20tckNoaudNoEnv/jvm.options

```diff
@@ -0,0 +1 @@
+-Dcom.ibm.tools.attach.enable=yes
```

Comment:
Please add `-Dcom.ibm.ws.beta.edition=true` to all jvm.options files in all the Liberty servers in this TCK so that it runs the beta protected code when ready. Thanks!